modules/gwlb/.header.md

```diff
@@ -0,0 +1,79 @@
+# Gateway Load Balancer Module for Azure
+
+A Terraform module for deploying a Gateway Load Balancer for VM-Series firewalls.
+
+## Usage
+
+In order to use GWLB, below minimal definition of Gateway Load Balancer can be used, for which:
```

Comment:
Please follow the MD conventions, new line before a list

Reply:
Fixed